### ISSUE TITLE
D415play support

### ISSUE THREAD

**Issue by Vinky41** [cowboy/synology-update-plex#19]
Hi,

The DS415play it's not supported (version x86).

`FAIL: Unable to find release for DS415play-j/i686/i686`

I test on my NAS this code and it's worked :

```
function get_arch() {
  local arch machine=$1 hw_version=$2
  if [[ "$machine" =~ armv7 ]]; then
    declare -A model_machine_map
    model_machine_map[DS414j]=armv7hf_neon
    model_machine_map[DS115j]=armv7hf
    model_machine_map[RS815]=armv7hf
    model_machine_map[DS216se]=armv7hf
    model_machine_map[DS215jv10-j]=armv7hf_neon
    if [[ "${model_machine_map[$hw_version]+_}" ]]; then
      arch=${model_machine_map[$hw_version]}
    elif [[ "${hw_version//[^0-9]/}" =~ 1[5-8]$ ]]; then
      arch=armv7hf_neon
    else
      arch=armv7hf
    fi
  elif [[ "$hw_version" = 'DS415play-j' ]]; then
      arch=x86
  else
      arch=$machine
  fi
  echo $arch
}

```

The log :

```
Installed version: 1.18.5.2309-f5213a238

New version available!

[ Finding release ]
{
  "label": "Intel 32-bit (x10 Series, DS415play, and DS214play)",
  "build": "linux-x86",
  "distro": "synology",
  "url": "https://downloads.plex.tv/plex-media-server-new/1.19.4.2935-79e214ead/synology/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk",
  "checksum": "80b6bdae87587ada31f24eddeb963e7fa6dc23f3"
}

[ Downloading release package ]
2020-06-27 14:14:02 URL:https://downloads.plex.tv/plex-media-server-new/1.19.4.2935-79e214ead/synology/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk [100648960/100648960] -> "/tmp/plex.88gmdE/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk" [1]

[ Verifying checksum ]
Checksum valid!

[ Installing package ]
/tmp/plex.88gmdE/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk install successfully

[ Restarting Plex Media Server ]
package Plex Media Server start successfully

[ Cleaning up ]
Removing /tmp/plex.88gmdE

Done!

```

**Comment by cowboy** [comment]
Can you post what the output of the `uname -m` command and the contents of the `/proc/sys/kernel/syno_hw_version` file are?

**Comment by Vinky41** [comment]
Hi,

`uname -m` : `i686`

`/proc/sys/kernel/syno_hw_version` : `DS415play-j`

**Comment by cowboy** [comment]
I believe I've fixed this in https://github.com/cowboy/synology-update-plex/pull/20, can you give this version a try?

https://raw.githubusercontent.com/cowboy/synology-update-plex/x86/update-plex.sh

**Comment by Vinky41** [comment]
It's worked.

Thx ;)

```
Checking for a Plex Media Server update...

[ Enabling Plex Pass releases ]
Found Plex Token

[ Retrieving version data ]
Available version: 1.19.4.2935-79e214ead
Installed version: 1.18.5.2309-f5213a238

New version available!

[ Finding release ]
{
  "label": "Intel 32-bit (x10 Series, DS415play, and DS214play)",
  "build": "linux-x86",
  "distro": "synology",
  "url": "https://downloads.plex.tv/plex-media-server-new/1.19.4.2935-79e214ead/synology/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk",
  "checksum": "80b6bdae87587ada31f24eddeb963e7fa6dc23f3"
}

[ Downloading release package ]
2020-06-27 15:45:38 URL:https://downloads.plex.tv/plex-media-server-new/1.19.4.2935-79e214ead/synology/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk [100648960/100648960] -> "/tmp/plex.RMwidz/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk" [1]

[ Verifying checksum ]
Checksum valid!

[ Installing package ]
/tmp/plex.RMwidz/PlexMediaServer-1.19.4.2935-79e214ead-x86.spk install successfully

[ Restarting Plex Media Server ]
package Plex Media Server start successfully

[ Cleaning up ]
Removing /tmp/plex.RMwidz

Done!
```
